src-json/meta.json

```diff
@@ -49,6 +49,20 @@
 		"doc": "Allows [] access on an abstract",
 		"targets": ["TAbstract", "TAbstractField"]
 	},
+	{
+		"name": "AssemblyMeta",
+		"metadata": ":assemblyMeta",
+		"doc": "TODO: documentation",
```

Comment:
TODO? :)

Reply:
Ahah forgot to add this to my TODO list :D